examples/purchaseTesterTypescript/app/screens/HomeScreen.tsx

```diff
@@ -179,12 +180,38 @@ const HomeScreen: React.FC<Props> = ({
                   Show In-App messages
                 </Text>
             </TouchableOpacity>
+            <Divider />
             <TouchableOpacity
               onPress={() => navigation.navigate('Paywall')}>
               <Text style={styles.otherActions}>
                 Go to Paywall Screen
               </Text>
             </TouchableOpacity>
+            <TouchableOpacity
+              onPress={async () => {
+                const paywallResult = await RevenueCatUI.presentPaywallIfNeeded({
+                  requiredEntitlementIdentifier: 'pro_cat',
+                  // offeringIdentifier: 'default',
```

Comment:
Hmm once we add the `offeringIdentifier` parameter, this should be part of each offering instead? Or maybe it's ok to leave it here and not have a call with that parameter here, in which case, we could remove the comment I think?

Reply:
Oh like passing the whole offering? yes, I like that. I will remove the comment